### PR TITLE
Build dg.1 manpage

### PR DESCRIPTION
**Pull request by praiskup**
Fixes: #33

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "build_manpages"]
+	path = build_manpages
+	url = https://github.com/praiskup/build_manpages
```
Comment (TomasTomecek) [inline .gitmodules]: I wish it was a dedicated python package.

Comment (praiskup) [reply .gitmodules]: I don't plan to work on separate package now;  at least until we stabilize the argparse -> manpage convertor :).  Any other life-cycle than git-submodule would be to slow at this point.

**Comment by praiskup** [comment]
Thanks for the remarks.  I'll try to fix them in https://github.com/praiskup/build_manpages unless somebody is faster than me.

File: .gitmodules (hunk 1 above)
Comment (TomasTomecek) [reply .gitmodules]: I didn't meant to ask you to do the separate package, I was just sad with the distribution model. I'm glad it works for you as it is.

Comment (praiskup) [reply .gitmodules]: And I'm not against wrapping it as a real python package. +1 to any attempt towards this.

**Comment by praiskup** [comment]
The 'Name' section, I have no clue what should be used to fill the summary :( any idea is welcome.

The UNKNOWN value was filled, that is not issue in `build_manpages`.

**Comment by praiskup** [comment]
PTAL

**Comment by TomasTomecek** [comment]
I would use "description" from setup.py. I tried to hack it just now but realized the code of `build_manpage` is quite complicated. [This](https://github.com/praiskup/build_manpages/blob/master/build_manpages/build_manpage.py#L75) is what I mean (but when I play with it locally, it doesn't pick that code path).

**Comment by praiskup** [comment]
That's `distribution.description`, not `argparse.description` ;  and `distribution.description` is set per-project, not per-binary.  The `argparse.description` should already be put right behind the usage output (but not into the `NAME` field).

**Comment by TomasTomecek** [comment]
oh, right; that's a good point

PRs filled with suggested changes -- feel free to dismiss if you disagree.

**Comment by praiskup** [comment]
Meh, still -> the `DESCRIPTION` section is pretty standardized in man;  and since `description` in argparse might be multiline -- man:argparse is sort of 1:1 mapping.  Would you be against non-standard argparse argument, say:
   `parser.man_name_description`

Btw., good point with epilog.  That should be handled somehow.

**Comment by TomasTomecek** [comment]
> Would you be against non-standard argparse argument

no, do whatever you want :)

**Comment by praiskup** [comment]
PTAL

**Comment by TomasTomecek** [comment]
LGTM, well done!

**Pull request merged**